testprojects/src/java/org/pantsbuild/testproject/extra_jvm_options/BUILD

```diff
@@ -1,12 +1,18 @@
 jvm_binary(
-    extra_jvm_options = ['-Dproperty.color=orange', '-Dproperty.size=2', '-DMyFlag', '-Xmx1m'],
```

Comment:
Are the changes to this file necessary? If not, revert?

Reply:
Changing something under testprojects should in theory cause them to be covered by the integration test shards (the "test_testprojects_*" tests).

But adding an explicit test near your old one would be preferable.

Reply:
Not all the changes are necessary, but the addition of target `app_noopts` at the end is. The other changes are there for style consistency, but they can be reverted.